java/src/jmri/jmrit/simpleclock/SimpleClockFrame.java

```diff
@@ -561,6 +563,17 @@ private void setTimeSourceChanged() {
         if (index == internalSourceIndex) {
             clock.setInternalMaster(true, true);
```

Comment:
Here is the source of my confusion:

This line is only executed (per the `if` above), when the GUI element is currently selecting "internalSourceIndex".  It sets the internal master as "true".  I am _assuming_ that "internalSourceIndex" implies JMRI as master, and this _assumption_ seems supported by the statement `clock.setInternalMaster(true,true);`.

If your code changes _were intended_ to affect JMRI's operation _when acting as a fast clock "master"_, then, based on my _assumption_, I would expect your P/R changes to be in this "if" block, rather than in the "else" block (below).